db/seeds.rb

```diff
@@ -140,9 +140,9 @@
 end.save!
 
 TrafficCam.find_or_initialize_by(
-  title: "Broadway"
```

Comment:
Do we need to remove this traffic cam from the database manually?

Reply:
This is more for consistency in development. We don't have a good way to execute this in production so I usually manually run it.